src/features/import-maps.js

```diff
@@ -33,7 +33,7 @@ function processScripts () {
       script.sp = true;
       if (!script.src)
         return;
-      System.import(script.src.slice(0, 7) === 'import:' ? script.src.slice(7) : resolveUrl(script.src, baseUrl)).catch(function (e) {
+      systemJSPrototype.import(script.src.slice(0, 7) === 'import:' ? script.src.slice(7) : resolveUrl(script.src, baseUrl)).catch(function (e) {
```

Comment:
This change might break things, since it modifies the context of the `import` function. `systemJSPrototype` is not a systemjs instance and therefore won't have a module registry that is created in the constructor. See code below:

https://github.com/systemjs/systemjs/blob/98609dbeef01ec62447e4b21449ce47e55f818bd/src/system-core.js#L25

Reply:
SystemJS refers to a class, not to the global instance of the class. Only `System` is the instance and is what should be used when calling import